xml/System/Uri.xml

```diff
@@ -726,7 +728,9 @@ http://myUrl/
   
 ## Remarks  
  This constructor creates a new <xref:System.Uri> instance by combining an absolute <xref:System.Uri> instance, `baseUri`, with a relative <xref:System.Uri> instance, `relativeUri`. If `relativeUri` is an absolute <xref:System.Uri> instance (containing a scheme, host name, and optionally a port number), the <xref:System.Uri> instance is created using only `relativeUri`.  
-  
+ 
+ If the `baseUri` has relative parts (like `/api`), then the relative part must be terminated with a slash, (like `/api/`). 
+ 
```

Comment:
There's an additional complication with this constructor: if the base URI includes a relative part, it is replaced by the relative URI argument if the relative URI argument begins with a slash, indicating a relative URI from the root. Do you want to note that as well?